css/responsive.css

```diff
@@ -0,0 +1,135 @@
+@media all and (min-width: 768px) {
```

Comment:
There seems to be an issue with your desktop version of the CSS. Right after moving to the desktop version and up to a certain width, your portfolio has some misalignments and leaves a lot of spaces to the right.
- [ ] The **navbar** and **the banner** has some space to the right. Both of them should occupy the full width at all times. Kindly fix this.

- [ ] The listed works only align to the left, but they should be at the center of the page (leaving equal margins to the left and right at all times). Kindly fix this.

- [ ] The footer section also has some space to the right. It should cover the full width of the screen at all times.

  Navbar, banner, and listed works | Footer section
  :---:|:---:
  ![tugsem-html](https://user-images.githubusercontent.com/36542986/159883454-67fea4e1-a93b-4352-bf15-9bde515a7928.jpg) | ![tugsem-footer](https://user-images.githubusercontent.com/36542986/159883448-06b9e5b7-cbd6-4c27-8743-ac0ae1b0d017.jpg)